com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -370,30 +372,40 @@ private void OnDestroy()
             ReplNetworkState.OnValueChanged -= OnNetworkStateChanged;
         }
 
+        [ServerRpc(Delivery = RpcDelivery.Reliable)]
+        private void SubmitNetworkStateServerRpc(NetworkState networkState)
+        {
+            // as a server, apply whatever networkstate owner client sent to us, to make NetworkTransform move locally on the server
+            ApplyNetworkState(networkState);
+
+            // as a server, update netvar<networkstate> to cause it to be replicated down to clients
+            ReplNetworkState.Value = networkState;
+            ReplNetworkState.SetDirty(true);
```

Comment:
to refactor a little further, let's extract these two lines (and the other two lines below in the `FixedUpdate()` method) into `private void UpdateNetworkVariable(NetworkState networkState)`?

Reply:
I'm already doing this extraction in my interpolation PR

Reply:
errr. oops. Just committed. Guess this will result in small merge conflicts.

Reply:
let's focus on this PR individually and integration with interpolation later? I see this as a meta-comment in the PR.